toqito/state_metrics/bures_distance.py

```diff
@@ -0,0 +1,68 @@
+"""Bures distance metric."""
+import numpy as np
+
+from toqito.state_metrics import fidelity
+
+def bures_distance(rho_1: np.ndarray, rho_2: np.ndarray, decimals: int = 10) -> float:
+    r"""
+    Compute the Bures distance of two density matrices [WikBures]_.
+
+    Calculate the Bures distance between the two density matrices:
+    ":code:`rho_1` and ":code:`rho_2`, defined by:
+    .. math::
+        \sqrt{2 (1 - F(\rho_1, \rho_2)},
+
+    where :math:`F(\cdot)` denotes the fidelity between :math:`\rho_1` and :math:`\rho_2`.
+    The return is a value between :math:`0` and :math:`\sqrt(2)`,with
```

Comment:
You should alter the instances of \sqrt(2) to \sqrt{2} (note the `(` and `)` vs. the `{` and `}`.